bindings/gumjs/gumscriptcore.cpp

```diff
@@ -1332,6 +1344,35 @@ gum_script_core_on_native_pointer_to_json (
   info.GetReturnValue ().Set (String::NewFromUtf8 (isolate, buf));
 }
 
+/*
+ * Prototype:
+ * NativePointer.toMatchPattern(pointer)
+ *
+ * Docs:
+ * Represents the pointer as a pattern.
+ *
+ * Example:
+ * TBW
+ */
+static void
+gum_script_core_on_native_pointer_to_match_pattern (
+    const FunctionCallbackInfo<Value> & info)
+{
+  GumScriptCore * self = static_cast<GumScriptCore *> (
+      info.Data ().As<External> ()->Value ());
+  Isolate * isolate = self->isolate;
+
+  gsize ptr = GPOINTER_TO_SIZE (GUM_NATIVE_POINTER_VALUE (info.Holder ()));
+  gchar buf[32];
+  #if G_BYTE_ORDER == G_LITTLE_ENDIAN
+    sprintf (buf, "%0" G_STRINGIFY (GLIB_SIZEOF_VOID_P_IN_NIBBLE) G_GSIZE_MODIFIER "x", GSIZE_TO_BE (ptr));
+  #else
+    sprintf (buf, "%0" G_STRINGIFY (GLIB_SIZEOF_VOID_P_IN_NIBBLE) G_GSIZE_MODIFIER "x", GSIZE_TO_LE (ptr));
+  #endif
```

Comment:
It would be great to skip indenting the `#if`, and just let the `sprintf()` calls have two space indentations. Also, lines in C/C++ code of *frida-gum* should never exceed 80 cols, and the continuation line should have 4 space indentation.